### DISCUSSION TITLE
Status Bericht Iteration 10

### DISCUSSION THREAD

**Discussion by CodeRider101** [SE-TINF22B2/G2-Schoolify#151] [Status Berichte]
### ✅Was haben wir geschafft?
- Issues aus den User Stories definiert -> #93 
- Architektur: Componenten-Diagramm fertiggestellt
- Neue Label eingeführt
   - Phase 0 -> Kann jetzt angefangen werden
   - Phase 1 -> kann direkt nach 0 umgesetzt werden
   - Phase 2 -> wird im späteren Verlauf bearbeitet
- Issues in 'optional' oder 'must have' einkategoriert
   - optional: 
       - #18 
       - #146 
       - #132 & #133 
       - #101 & #104 
       - Allgemeine Suche von Personen in der Schule

### 🏆Was sind unsere Ziele für diese Woche und ggf. kommenden Wochen?
- ERM überarbeiten und nochmal reviewen
- Fragen:
   - Welche Aufgaben sind erledigt - welche stehen an?
   - Wie liegen wir in der Zeit?
   - Neuer Termin für die Meetings?

### 🛠️Wie sind wir vorgegangen und gibt es Verbesserungsvorschläge?
Agenda am Anfang der Woche eingesendet. Agenda auf Wünsche überarbeitet (Themen hinzufügen). Blocker in den Arbeitskalender gesetzt, damit die Ausbilder wissen man ist zur Zeit verplant. Agenda besprochen. 30 Minuten länger als geplant (insgesamt eine Stunde). Weiteres auf nächstes Meeting verschoben.

Verbesserungsvorschläge:
- große Themen die wichtig sind in ein separates Meeting auslagern, damit Agenda komplett durchlaufen werden kann
- nicht immer möglich, da von der Arbeit aus Meetings anstehen und manche verhindert sind

**Comment by roland-KA** [comment]
Freut mich, dass es weiter vorangeht 😊👍
